Sources/SwiftParser/Attributes.swift

```diff
@@ -376,7 +376,10 @@ extension Parser {
       }
     case nil:
       return parseAttribute(argumentMode: .customAttribute) { parser in
-        let arguments = parser.parseArgumentListElements(pattern: .none, allowTrailingComma: false)
+        let arguments = parser.parseArgumentListElements(
+          pattern: .none,
+          allowTrailingComma: parser.experimentalFeatures.contains(.trailingComma)
+        )
```

Comment:
I think you should probably also allow trailing commas in all the custom attributes above, like `@_backDeployed`, `@differentiable` etc.

Reply:
Thanks for adding them.